export/http.go

```diff
@@ -0,0 +1,87 @@
+package export
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"regexp"
+
+	"go.opencensus.io/metric/metricdata"
+	"go.opencensus.io/resource"
+	"google.golang.org/protobuf/proto"
+)
+
+// HTTP is an exporter that exports metrics to an
+// HTTP endpoint
+type HTTP struct {
+	address   string
+	aPIkey    string
+	aPISecret string
```

Comment:
the capitalization in here is a bit funny. how about `apiKey` and `apiSecret` instead?